config/webpack/config.base.js

```diff
@@ -42,7 +43,7 @@ module.exports = ({ sourceDir, distDir }) => ({
             options: {
               name: "[name].[ext]",
               outputPath: "fonts/",
-              publicPath: "/fonts/",
+              publicPath: STATIC_URL+"fonts/",
```

Comment:
You don't have to concatenate strings here - it's sufficient to just put `/fonts/` here

Reply:
ok, I solved this on my branch.